### DISCUSSION TITLE
More quick options

### DISCUSSION THREAD

**Discussion by Blubmann1337** [nilsteampassnet/TeamPass#3996] [Ideas]
Hi,
it would be useful if further quick options were possible for the individual entries so that you don't always have to go into an item. In addition to "open", "edit", "open website", etc., things like copy user name, copy password and display password (perhaps as a popup) would be good.
![image](https://github.com/nilsteampassnet/TeamPass/assets/80953608/4d6bcae2-0f73-4719-bf4c-828c28fee871)

**Comment by nilsteampassnet** [comment]
@Blubmann1337 
Thank you for your input.
You can add a quick icon for 'copy pwd' and 'copy username' through dedicated setting in options.
![image](https://github.com/nilsteampassnet/TeamPass/assets/1197546/1cd8d5b4-39c6-4ee9-84f3-02941e9fa930)

**Reply by Blubmann1337** [reply]
Oh yes, damn I'm stupid. Thanks for the hint
